cgmanifest.json

```diff
@@ -8301,8 +8301,8 @@
         "type": "other",
         "other": {
           "name": "kernel-mshv",
-          "version": "6.6.57.mshv4",
-          "downloadUrl": "https://azurelinuxsrcstorage.blob.core.windows.net/sources/core/kernel-mshv-6.6.57.mshv4.tar.gz"
+          "version": "6.6.100.mshv1",
+          "downloadUrl": "https://github.com/microsoft/CBL-Mariner-Linux-Kernel/archive/refs/tags/rolling-lts/kata/6.6.100.mshv1.tar.gz"
```

Comment:
nit: the cgmanifest check is failing because the source0 doesn't match the file. Can you update the url in the cgmanifest to 
https://github.com/microsoft/CBL-Mariner-Linux-Kernel/archive/rolling-lts/kata/6.6.100.mshv1.tar.gz
instead of 
https://github.com/microsoft/CBL-Mariner-Linux-Kernel/archive/**refs/tags**/rolling-lts/kata...

Reply:
Updated. Thanks!